src/summary_statistics/tmrca.cc

```diff
@@ -22,13 +22,14 @@
 #include "tmrca.h"
 
 void TMRCA::calculate(const Forest &forest) {
-  if (forest.calcSegmentLength() == 0) return;
+  //std::cout << "forest.calcSegmentLength() "<<forest.calcSegmentLength() <<std::endl;
+  if (forest.rec_bases_.size() == 1) return;
```

Comment:
The first line wasn't wrong. When I was checking for the copy constructor and assignment operator, I was using valgrind on the unit_test, which complained about the uninitialised values on that line.